beacon-chain/core/feed/state/BUILD.bazel

```diff
@@ -7,6 +7,9 @@ go_library(
         "notifier.go",
     ],
     importpath = "github.com/prysmaticlabs/prysm/beacon-chain/core/feed/state",
-    visibility = ["//beacon-chain:__subpackages__"],
+    visibility = [
+        "//beacon-chain:__subpackages__",
+        "//validator:__subpackages__",
+    ],
```

Comment:
What's up with this? `//beacon-chain/core/feed/state:go_default_library` isn't used in these changes on the validator side.

Reply:
Removed. Thank you